deploy/airgap-prepare.sh

```diff
@@ -111,9 +92,6 @@ rm $K3S_INSTALL_SCRIPT \
 	$K3S_BINARY \
 	$K3S_IMAGES_TAR \
 	$CRED_SHIELD_IMAGES_TAR \
-	$K3S_SELINUX_MICROOS \
-	$K3S_SELINUX_CENTOS7 \
-	$K3S_SELINUX_CENTOS8 \
 	$CERT_MANAGER_IMAGES_TAR \
```

Comment:
Remove the policy install.sh and rego files from the rpm package.

Reply:
Done